cypress-io/cypress
# fix: The record key is obstructed in v10

- closes https://github.com/cypress-io/cypress/issues/22091

### User facing changelog
Fixed a bug where the user can select just the key, need to select the entire command using the button, and then trim it to use just the key

### Additional details
Setting up the Dashboard recording using v10.0.2 and cannot select just the key, need to select the entire command using the button and then trim it to use just the key

### Steps to test
- Add dashboard to any project.
- See the window, prompting you to run the provided command in your local or CI environment.
- Verify that you can select/highlight a specific portion of the provided command.
- Verify that you can pan around (scroll) horizontally in case of command overflow.


### How has the user experience changed?
BEFORE

https://user-images.githubusercontent.com/45232708/194007590-65570315-4969-4182-a3af-00987707f9d4.mov


AFTER

https://user-images.githubusercontent.com/45232708/194009673-e3e07f79-246c-4c5f-942d-9a46ae9ec46b.mov

### PR Tasks
